csharp/ql/src/semmle/code/csharp/controlflow/ControlFlowGraph.qll

```diff
@@ -108,8 +108,8 @@ module ControlFlow {
     /**
      * Holds if this node post-dominates `that` node.
      *
-     * That is, all paths reaching a callable exit node (`ExitNode`)
-     * from `that` node must go through this node.
+     * That is, all paths reaching a normal callable exit node (an `AnnotatedExitNode`
+     * with a normal normal exit type) from `that` node must go through this node.
```

Comment:
typo: Should be `with a normal exit type` instead of `with a normal normal exit type`.